doc/configuration.md

```diff
@@ -0,0 +1,43 @@
+## Configuration
+
+Amoc is configured through OTP application environment variables that
+are loaded from the configuration file, operating system environment variables
+(with prefix ``AMOC_``) and Erlang application environment variables
+(`priv/app.config`).
+
+`AMOC_HOSTS` (the hostnames of all of the `amoc` machines) must be set in order to successfully
+run the scenario in distributed environment.
+
+Amoc is able to report metrics to Graphite.
+It is possible to set up reporting endpoint by setting `AMOC_GRAPHITE_HOST`
+and `AMOC_GRAPHITE_PORT` enviornmental variables.
+
+If there is a need to point amoc to some additional paths with code,
+it can be done by specifying variable `AMOC_EXTRA_CODE_PATHS`.
+
+Internally, amoc is using the following settings:
+
+- ``interarrival`` - a delay in ms, for each node, between creating process
```

Comment:
> [...] between creating processes [...]

In plural. I understand that it is about processes per user, right?

Reply:
Yup